geomstats/backend/numpy/__init__.py

```diff
@@ -140,14 +140,42 @@ def get_mask_i_float(i, n):
 
 
 def assignment(x, values, indices, axis=0):
```

Comment:
Could you explain why assignment is complicated in numpy? I'm just worried that we will loose a lot in performances if this part is not optimized.

For example, `x` shouldn't be copied, it should be modified, right?

Reply:
The problem is that, since Tensorflow does not allow modifying (or at least, I don't know how), if we want behaviors to be as similar as possible, this is needed. In some cases (weird, but not that uncommon), having backends which modify an object or create a new one might lead to completely different outcomes, for instance in lists.
However, there's also the possibility of specifying in the doc that behaviors differ, asking them to be cautious.